src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -3781,18 +3872,18 @@ def main():
             event.accept()
         # Call the appropriate handler
         if hooks.invoke_handler(event, cgroup, jobutil) is True:
-            pbs.logmsg(pbs.EVENT_DEBUG2, "%s: Hook handler returned success" %
+            pbs.logmsg(pbs.EVENT_DEBUG4, "%s: Hook handler returned success" %
                        (caller_name()))
             event.accept()
         else:
             pbs.logmsg(pbs.EVENT_DEBUG, "%s: Hook handler returned failure" %
                        (caller_name()))
             event.reject()
     except SystemExit:
-        # The e.accept() and e.reject() methods generate a SystemExit
+        # The event.accept() and event.reject() methods generate a SystemExit
```

Comment:
By doing a pass here, aren't you ignoring the SystemExit and therefore ignoring the accept() or reject()?

Reply:
That doesn't seem to be the case in practice. This was something that was added early on and has not changed for a long time. We want the hook to exit with a zero return code, and catching SystemExit allows that to happen.